docs/index.rst

```diff
@@ -1,31 +1,61 @@
-=======================
-Scylla Care-Pet Example
-=======================
+.. title:: Scylla Care-Pet Example
 
-.. toctree::
-   :maxdepth: 2
-   :hidden:
+.. raw:: html
 
-   Getting Started with CarePet: A sample IoT App <getting_started>
-   Design and Data Model <design_and_data_model>
-   Care-Pet GitHub Repository <https://github.com/scylladb/care-pet>
-   Care-Pet Blog <https://www.scylladb.com/2020/09/09/carepet-an-example-iot-use-case-for-hands-on-app-developers/>
+  <div class="landing landing--floating">
+
+.. hero-box::
+  :title: Scylla Care-Pet Example
+  :button_icon: fa fa-github
+  :button_url: https://github.com/scylladb/care-pet
+  :button_text: Care-Pet Project
+
+  This example project demonstrates a generic IoT use case for ScyllaDB.
+
+.. raw:: html
+
+  <div class="landing__content landing__content">
 
+.. raw:: html
 
-This example project demonstrates a generic IoT use case for ScyllaDB.
+  <div class="topics-grid grid-container full">
 
-The application allows tracking of pets health indicators and consist of 3 parts:
+  <div class="grid-x grid-margin-x">
 
-* a collar that reads and pushes sensors data
-* a web app for reading and analyzing the pets' data
-* a database migration tool
+.. topic-box::
+  :title: Getting Started with Care-Pet
+  :link: getting_started
+  :icon: scylla-icon scylla-icon--getting-started
+  :class: large-6
+  :anchor: Get Started
 
-Pick a topic to get started:
+  Create an IoT app from scratch and configure it to use ScyllaDB as the backend datastore.
 
-* `Getting Started with CarePet: A sample IoT App <getting_started>`_
+.. topic-box::
+  :title: Design and Data Model
+  :link: design_and_data_model
+  :icon: scylla-icon scylla-icon--overview
+  :class: large-6
+  :anchor: Learn more
 
-* `Design and Data Model <design_and_data_model>`_
+  Learn more about the queries used in the example, make the primary key and clustering key selection, and create the database schema.
+
+.. topic-box::
+  :title: Care-Pet Blog
+  :icon: fa fa-newspaper
+  :link: https://www.scylladb.com/2020/09/09/carepet-an-example-iot-use-case-for-hands-on-app-developers/
+  :class: large-6,small-12
+  :anchor: Read the article
+
+  Read our blog!
+
```

Comment:
```suggestion

.. raw:: html

  </div></div></div></div>
  
  ````

Reply:
@dgarcia360 why do we need this? can we use the box directive instead?